modules/developers-guide/examples/javascript/node-full-script.mjs

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env zx
+
+import * as grpc from "@grpc/grpc-js";
+import { StargateClient, StargateTableBasedToken, Query, Batch, BatchQuery, Response, promisifyStargateClient } from "@stargate-oss/stargate-grpc-node-client";
+
+try {
+    // Stargate OSS configuration for locally hosted docker image
+    const auth_endpoint = "http://localhost:8081/v1/auth";
+    const username = "cassandra";
+    const password = "cassandra";
+    const stargate_uri = "localhost:8090";
+
+    // Set up the authenication
+    // For Stargate OSS: Create a table based auth token Stargate/Cassandra authentication using the default C* username and password
+    const credentials = new StargateTableBasedToken({authEndpoint: auth_endpoint, username: username, password: password});
```

Comment:
Nit: in Node/JS if you're creating an object where the key and value names are the same, you can omit declaring them twice. So here we could do `{authEndpoint: auth_endpoint, username, password}`. Either way works, so not a priority.

Reply:
Good to know - I think I'll leave it as written, to be crystal clear. But thanks for educating me. :-)

Reply:
good idea - leaving as is for clarity